modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -204,7 +204,7 @@ $ ccoctl gcp create-all \
   --project=<gcp_project_id> \// <3>
   --credentials-requests-dir=<path_to_credentials_requests_directory> <4>
 ----
-<1> Specify the user-defined name for all created GCP resources used for tracking.
+<1> Specify the user-defined name for all created {gcp-short} resources used for tracking. You must provide the identity pool ID and provider ID in the format of `<EXAMPLE>`. If you plan to install the {gcp-short} Filestore Container Storage Interface (CSI) Driver Operator, retain these values.
```

Comment:
The pool ID and provider ID are optional values, not required for CSI operator. They will both default to the name, which can then be used when installing the operator. I think the comment to make here is to take note of the value provided here as it may be required when installing additional operators.

Reply:
So, probably:
```suggestion
<1> Specify the user-defined name for all created {gcp-short} resources used for tracking. If you plan to install the {gcp-short} Filestore Container Storage Interface (CSI) Driver Operator, retain this value.
```